docs/deploy/linux/os-native/upgrade.md

```diff
@@ -275,10 +275,10 @@ sudo yum clean all
 
 ```shell
 sudo tee /etc/zypp/repos.d/rocm.repo <<EOF
-[ROCm-5.5.1]
-name=ROCm5.5.1
+[ROCm-5.6]
```

Comment:
Let's use 3 digits always as the repository isn't really a 5.6 series but just 5.6.0

Reply:
This would be inconsistent with the OS native install guide as it uses a loop to generate the repository entry with URL.

```shell
for ver in 5.3.3 5.4.3 5.5.1 5.6; do
sudo tee --append /etc/zypp/repos.d/rocm.repo <<EOF
[ROCm-$ver]
name=ROCm$ver
...
```